data/event-schemas/schema/m.call.negotiate.yaml

```diff
@@ -64,6 +64,23 @@ properties:
           Once the invite age exceeds this value, clients should discard it.
           They should also no longer show the call as awaiting an answer in the
           UI.
+      sdp_stream_metadata:
+        type: object
+        title: SDP Stream Metadata
+        x-addedInMatrixVersion: '1.7'
+        description: Object describing the streams that will be sent
```

Comment:
should probably link to https://pr1602--matrix-spec-previews.netlify.app/client-server-api/#streams, since there is insufficient documentation inline